### PR TITLE
Translation - Improve French

### PR DESCRIPTION
**Pull request by Alfred-Neuman**

### CODE REVIEW

File: addons/airway/stringtable.xml [hunk 1]
```diff
@@ -201,6 +201,7 @@
         <Key ID="STR_KAT_Airway_checkAirway_poisonLog">
             <English>%1 checked airways: Heavily Blistered</English>
             <Japanese>%1 が気道を確認: ひどい水膨れ</Japanese>
+            <French>%1 voies aériennes contrôlées : Fortement cloqué</French>
```
Comment (Fanrod-Bones) [inline addons/airway/stringtable.xml]: Fortement cloquées

File: addons/airway/stringtable.xml [hunk 2]
```diff
@@ -410,6 +411,7 @@
         <Key ID="STR_KAT_Airway_AirwayStatus_Poison">
             <English>Airway is heavily blistered</English>
             <Japanese>気道はひどく水膨れしている</Japanese>
+            <French>Les voies respiratoires sont fortement boursouflées</French>
```
Comment (Fanrod-Bones) [inline addons/airway/stringtable.xml]: fortement cloquées

File: addons/watch/stringtable.xml [hunk 3]
```diff
@@ -158,31 +158,37 @@
...
         <Key ID="STR_KAT_Watch_AddTimer_Setting">
             <English>Add 15 Seconds to Watch Timer</English>
             <Japanese>時計のタイマーを15秒増やす</Japanese>
+            <French>Ajouter 15 secondes à la minuterie de la montre</French>
             <Chinesesimp>添加15秒观看计时器</Chinesesimp>
         </Key>
         <Key ID="STR_KAT_Watch_SubtractTimer_Setting">
             <English>Subtract 15 Seconds from Watch Timer</English>
             <Japanese>時計のタイマーを15秒減らす</Japanese>
+            <French>Retirez 15 secondes de la minuterie de la montre</French>
```
Comment (Fanrod-Bones) [inline addons/watch/stringtable.xml]: Retirer

**Review by Fanrod-Bones — CHANGES_REQUESTED** [review]
Just a few things but otherwise everything lgtm thank yoooou

**Comment by Alfred-Neuman** [comment]
fixed not without difficulty
thanks to @Fanrod-Bones and @MiszczuZPolski

**Pull request merged**